Docs/index.md

```diff
@@ -40,7 +40,7 @@ llbuild2 is architected around the 2 phases that happen in any build system impl
 
 llbuild2 makes heavy use of CAS (Content Addressable Storage) technologies. With CAS, data, file and directory
 structures can be represented and accessed by the digest (or hash) of its contents. Using CAS identifiers, llbuild2 can
-detect when changes have ocurred to any portion of the build graph, and only reëvaluate the pieces that have never been
+detect when changes have occurred to any portion of the build graph, and only reevaluate the pieces that have never been
```

Comment:
I always have trouble with "occurred", I actually wrote it right the first time and changed it to this..

but I disagree on the ë change: https://www.newyorker.com/culture/culture-desk/the-curse-of-the-diaeresis

Reply:
TIL re: the diaeresis